_components/headers.md

```diff
@@ -5,8 +5,453 @@ title: Headers & Navigation
 ---
 
 <div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/HeaderNav_FullUI_v1-930width.png">
+
+  <h2>Header Nav Simple</h2>
+
+  <header role="banner">
+    <div class="usa-grid">
+      <div class="usa-nav">
+        <a class="usa-menu-btn" id="menu-btn" href="#">Menu</a>
+        <div class="usa-logo-block" id="logo">
+          <a href="javascript:void(0)" accesskey="1" aria-label="Home">
+            <h1 class="usa-logo-title">Department of Web Standards</h1>
+          </a>
+        </div>
+        <a class="usa-search-trigger" href="#">
+        </a>
+        <nav class="usa-nav-primary" role="navigation">
+          <ul class="usa-nav-list usa-unstyled-list">
```

Comment:
@msecret do you think we should go through the library and do the same thing for all the places we've used the `usa-unstyled-list` class?

I think this is much cleaner markup.